code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1027 @@
+	else
+		user.visible_message(span_warning("[user] screws up, damaging [their_or_other] [limb.plaintext_zone] in their efforts to help!"))
+		limb.receive_damage(brute = 5, damage_source = user, wound_bonus = CANT_WOUND)
+
+	if (HAS_TRAIT(user, TRAIT_RESISTHEAT) || HAS_TRAIT(user, TRAIT_RESISTHEATHANDS))
+		return
+
+	to_chat(user, span_warning("You burn your hand on [victim]'s [limb.plaintext_zone]!"))
+	var/obj/item/bodypart/affecting = user.get_bodypart("[(user.active_hand_index % 2 == 0) ? "r" : "l" ]_arm")
+	affecting?.receive_damage(burn = 5)
+
+// T2 burn wounds are required to mold metal, which finished the first step of treatment. Aggrograb someone and use a welder on them for a guaranteed wound with no damage
+/datum/wound/blunt/robotic/critical/proc/heat_metal(obj/item/welder, mob/living/user)
+	if (!welder.tool_use_check())
+		return TRUE
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	if (user)
+		user.visible_message(span_notice("[user] carefully holds [welder] to [their_or_other] [limb.plaintext_zone], slowly heating it..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if (!welder.use_tool(target = victim, user = user, delay = 10 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+
+	var/wound_path = /datum/wound/burn/robotic/overheat/severe
+	if (user != victim && user.combat_mode)
+		wound_path = /datum/wound/burn/robotic/overheat/critical
+		user.visible_message(span_warning("[user] heats [victim]'s [limb.plaintext_zone] aggressively, overheating it far beyond the necessary point!"))
+		limb.receive_damage(burn = 10, damage_source = welder)
+
+	var/datum/wound/burn/robotic/overheat/overheat_wound = new wound_path
+	overheat_wound.apply_wound(limb, wound_source = welder)
+
+	to_chat(victim, span_green("Your [limb.plaintext_zone] is now heated, allowing it to be molded! Your next step is to have someone physically reset the superstructure with their hands."))
+	return TRUE
+
+// An RCD can be used on a T3 wound to finish its 1st treatment step with little risk and no burn wound
+/datum/wound/blunt/robotic/critical/proc/rcd_superstructure(obj/item/construction/rcd/treating_rcd, mob/user)
+	if (!treating_rcd.tool_use_check() || treating_rcd.get_matter(user) < ROBOTIC_T3_BLUNT_WOUND_RCD_COST)
+		return TRUE
+
+	if (user)
+		user.visible_message(span_notice("[treating_rcd] whirs to life as it begins replacing the damaged superstructure of [victim]'s [limb.plaintext_zone]..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if (!treating_rcd.use_tool(target = victim, user = user, delay = 10 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+	treating_rcd.useResource(ROBOTIC_T3_BLUNT_WOUND_RCD_COST, user)
+
+	var/chance = 100
+	if (victim == user)
+		chance *= 0.75
+
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
```

Comment:
Why are there x2 and x5 multipliers if the base chance is already 100% and the minimum chance you could have is 75%?